tools/checked-c-convert/CheckedCConvert.cpp

```diff
@@ -470,13 +474,26 @@ class RewriteConsumer : public ASTConsumer {
           DS = K->second;
         
         PVConstraint *PV = nullptr; 
-        for (const auto &V : Vars) 
-          if(PVConstraint *T = dyn_cast<PVConstraint>(V))
+        FVConstraint *FV = nullptr;
+        for (const auto &V : Vars) {
+          if (PVConstraint *T = dyn_cast<PVConstraint>(V))
             PV = T;
+          else if (FVConstraint *T = dyn_cast<FVConstraint>(V))
+            FV = T;
+        }
 
         if (PV && PV->anyChanges(Info.getConstraints().getVariables())) {
+          // Rewrite a declaration.
           std::string newTy = PV->mkString(Info.getConstraints().getVariables());
           rewriteThese.insert(DAndReplace(DeclNStmt(D, DS), newTy));
+        } else if (FV && FV->anyChanges(Info.getConstraints().getVariables())) {
+          // Rewrite a function variables return value.
```

Comment:
Typo in function variables return value?  I think you need an apostrophe after variables.  Alternately, you could write "return value of a function variable".